egs2/slue-voxpopuli/slu1/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train"
+valid_set="devel"
+test_sets="test devel"
+
+slu_config=conf/tuning/train_asr_no_pretrain.yaml
```

Comment:
Consider make it standardize by setting conf/train.yaml

Reply:
Done!